exporter/trace/stackdriver/stackdriver.go

```diff
@@ -16,14 +16,13 @@
 //
 // Example:
 //
-// 	import "go.opencensus.io/trace/adaptor/stackdriver"
+// 	import "go.opencensus.io/exporter/trace/stackdriver"
```

Comment:
Also please add "go.opencensus.io/trace" lest the example still hangs so
```go
//  import (
//        "go.opencensus.io/exporter/trace/stackdriver"
//        "go.opencensus.io/trace"
//  )
```

Reply:
Done, thanks.